src/FSharpLint.Core/Rules/Conventions/Binding/FavourIgnoreOverLetWild.fs

```diff
@@ -20,7 +20,7 @@ let private checkForBindingToAWildcard pattern range fileContent (expr: SynExpr)
               Message = Resources.GetString("RulesFavourIgnoreOverLetWildError")
               SuggestedFix = Some (lazy (Some({ FromRange = letBindingRange
                                                 FromText = fileContent
-                                                ToText = exprText + " |> ignore" })))
+                                                ToText = ("(" + exprText + ")" + " |> ignore").Trim() })))
```

Comment:
`.Trim()` does nothing here and whole expression can be rewritten as `sprintf "(%s) |> ignore" exprText`